Resources/Textures/_Nuclear14/Structures/Storage/DumpsterH.rsi/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "converted by Marines for use in nuclear14",
+    "size": {
+      "x": 64,
+      "y": 48
+    },
+    "states": [
+      {
+        "name": "DumpsterH",
+        "directions": 2
```

Comment:
Not sure if 2 is a recognised direction, may need to be 4 with two duplicated states, we shall see.